deployability/modules/testing/tests/helpers/central.py

```diff
@@ -19,23 +19,33 @@ def install_aio(inventory_path, wazuh_version) -> None:
             wazuh_version (str): major.minor.patch
 
         """
-        wazuh_version = '.'.join(wazuh_version.split('.')[:2])
         os_name = HostInformation.get_os_name_from_inventory(inventory_path)
 
+        if live == "False":
+            s3_url = 'packages-dev.wazuh.com'
+        else:
+            s3_url = 'packages.wazuh.com'
+
```

Comment:
Please use a boolean

Reply:
In principle it was contemplated to be boolean and it was treated as such, but there is a problem with the booleans since the `pytest` are invoked by Ansible and the passage of variables through Ansible until reaching the test, causes it to pass the booleans as strings. This problem requires removing Ansible and running the `pytest` directly, as it makes no sense to use Ansible to run from localhost. But it is not part of this issue, as it requires more effort.

Reply:
We need to refactor this module, it will be done in DTT2